PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -10,34 +10,43 @@ function New-Report {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [ValidateSet("Teams", "EXO", "Defender", "AAD", "PowerPlatform", "SharePoint", "OneDrive", IgnoreCase = $false)]
```

Comment:
Is it okay these are mixed case with ignoreCase false and in Orchestrator they are all lower case?